source/_integrations/sentry.markdown

```diff
@@ -0,0 +1,29 @@
+---
+title: "Sentry"
+description: "Record errors to Sentry."
+logo: sentry.png
+---
+
+The `sentry` integration integrates with [Sentry](https://sentry.io/) to capture both logged errors as well as unhandled exceptions in Home Assistant.
+
+## Configuration
+
+To use the `sentry` integration in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+sentry:
+  dsn: SENTRY_DSN
+  environment: development # optional
```

Comment:
We only want required config variables in the first example, so please remove `environment`

Reply:
Sorry thought I fixed this